mythril/laser/ethereum/svm.py

```diff
@@ -63,6 +66,7 @@ def sym_exec(self, main_address):
         self._sym_exec()
 
         logging.info("Execution complete")
+        logging.info("Achieved {0:.3g}% coverage".format(self.coverage))
```

Comment:
this won't be visible to the user right?

Reply:
If you do -v1 or v2 then you can see it. This pr was only about the functionality, not really about presentation